sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHost.cs

```diff
@@ -21,6 +21,8 @@ internal class EventProcessorHost : EventProcessor<EventProcessorHostPartition>
         private BlobCheckpointStoreInternal _checkpointStore;
 
         private ConcurrentDictionary<string, CheckpointInfo> _lastReadCheckpoint = new();
+        private EventHubConnection _connection;
```

Comment:
I don't know that I understand the purpose that it is intended to serve.   

Each partition owned by the client has its own unique connection.   Those are disposed when partition ownership is lost or surrendered.   There's no benefit to creating and capturing a single connection at the processor level just to have something to dispose.  It will have been closed/disposed when the processor is stopped.